src/block/button/style.scss

```diff
@@ -13,3 +13,7 @@
 .stk-button {
 	box-sizing: border-box;
 }
+
+.stk-block-button-group--flex-wrap .stk-block-button.stk-button--full-width {
+	margin-inline-end: unset !important;
```

Comment:
Do not use `!important` here, instead, make sure that you're selector is more specific than the one you're trying to override: `.stk-button-group [data-block]:not(:last-child) .stk-block`